src/address.ts

```diff
@@ -91,7 +92,7 @@ export class Address {
   }
 
   static UnmarshalPB(pb: pb.Address): Address {
-    return new Address(pb.getChainId(), new LocalAddress(pb.getLocal_asU8()))
+    return new Address(pb.getChainId(), new LocalAddress(Buffer.from(pb.getLocal_asU8())))
```

Comment:
I'm thinking we should do the `Buffer.from` in the `LocalAddress` constructor rather than here, that way the `LocalAddress.bytes` will always be the expected type, regardless of how the `LocalAddress` is created.

Reply:
Fixed, thank you for your review 👍